README.md

```diff
@@ -0,0 +1,80 @@
+![Wagtail](https://releases.wagtail.io/wagtail-github-header.png?x)
+
+Wagtail is an open source content management system built on Django, with a strong community and commercial support. It's focused on user experience, and offers precise control for designers and developers. 
+
+![Wagtail screenshot](https://releases.wagtail.io/wagtail-screenshot-with-browser.png)
```

Comment:
Love the new screenshot! I'd suggest using a simpler browser bar – without as many buttons. Again not a big deal but it's distracting when looking at the screenshot from top to bottom.

Reply:
Agreed, I should have thought of that. I'll re-do.